gradle-plugin/src/main/java/com/google/cloud/tools/dependencies/gradle/LinkageCheckTask.java

```diff
@@ -109,7 +110,7 @@ private boolean findLinkageErrors(Configuration configuration) throws IOExceptio
 
     ClassPathResult classPathResult =
         createClassPathResult(configuration.getResolvedConfiguration());
-    ImmutableList.Builder<ClassPathEntry> classPathBuilder = ImmutableList.builder();
+    ImmutableList.Builder<ClassPathEntry> classPathEntryBuilder = ImmutableList.builder();
```

Comment:
classPahBuilder was correct. ClassPathEntry is the element type, not the list type.

Reply:
Good point. Updated to `classPathEntriesBuilder`.